eng/Packages.Data.props

```diff
@@ -104,9 +104,9 @@
     <!-- Other approved packages -->
     <PackageReference Update="Microsoft.Azure.Amqp" Version="2.5.10" />
     <PackageReference Update="Microsoft.Azure.WebPubSub.Common" Version="1.1.0" />
-    <PackageReference Update="Microsoft.Identity.Client" Version="4.39.0" />
-    <PackageReference Update="Microsoft.Identity.Client.Desktop" Version="4.39.0" />
-    <PackageReference Update="Microsoft.Identity.Client.Extensions.Msal" Version="2.19.3" />
+    <PackageReference Update="Microsoft.Identity.Client" Version="4.46.0" />
+    <PackageReference Update="Microsoft.Identity.Client.Broker" Version="4.46.0-preview" />
```

Comment:
Did we intend to take a dependency on a preview package?

Reply:
Yes, `Microsoft.Identity.Client.Broker` is currently only available as a preview, and it is only referenced from the `Azure.Identity.BrokeredAuthentication` which is also in preview (beta in our terms). I wasn't sure if there was a way to denote this in Packages.Data.props. I suppose if nothing else I could add a comment to this effect.